### PR TITLE
#4 feat functionality cue cards translation

### LINKED ISSUE #7: - **#4** Functionality of cue cards
- [x] Give the user the option to reveal the englisch translation by clicking a button on the cue card.

### PR DESCRIPTION
**Pull request by JannisHeyer**
- Added an animation revealing the translation of the german vocabulary on the backside of the cue card.
- Fixed an issue that prevented the production build from being finalised.
- Tidied up the styles in the card component.
- Some tweaks based on the last code review.

### CODE REVIEW

File: components/Navbar.jsx [hunk 1]
```diff
@@ -7,17 +7,24 @@ export const Navbar = () => {
...
+              <img src="/Icons/AddCard-Black.svg" alt="Main Icon" />
+            </a>
           </Link>
         </li>
         <li>
-          <Link href="/components/Categories">
-            <img src="/Icons/Categories-Black.svg" alt="Main Icon" />
+          <Link href="/categories">
+            <a>
+              {" "}
```
Comment (VB2webD) [inline components/Navbar.jsx:25]: remove?

Comment (VB2webD) [inline components/Navbar.jsx]:
```suggestion
             
```

File: components/Navbar.jsx [hunk 2]
```diff
@@ -7,17 +7,24 @@ export const Navbar = () => {
       <ul>
         <li>
           <Link href="/">
-            <img src="/Icons/MainCueCard-Black.svg" alt="Main Icon" />
+            <a>
+              <img src="/Icons/MainCueCard-Black.svg" alt="Main Icon" />
```
Comment (VB2webD) [inline components/Navbar.jsx:11]: maybe use ["Image Component from next?"](https://nextjs.org/docs/api-reference/next/image)

File: styles/globalStyles.jsx [hunk 3]
```diff
@@ -6,6 +6,11 @@ const GlobalStyles = createGlobalStyle`
 --button-border-radius: 0.3rem;
 --button-height: 2.5rem;
 --button-width: 6.5rem;
+--card-height: 455.13px;
```
Comment (VB2webD) [inline styles/globalStyles.jsx:9]: maybe insert comments for Structure to keep a better overview as the project grows` eg  // Card Styling`

**Review by VB2webD — COMMENTED** [review]
LGTM

File: components/Card.jsx [hunk 4]
```diff
@@ -1,47 +1,103 @@
...
+        animate={isFlipped ? "back" : "front"}
+        variants={variants}
+      >
+        {!isFlipped ? (
+          <StyledCardContainer>
+            <h2>{article}</h2>
+            <h1>{word}</h1>
+            <p>{wordType}</p>
+            <p>{ipa}</p>
+            <h5>Category:</h5>
```
Comment (CultOfTheDeadCrow) [inline components/Card.jsx:32]: You are not using h1 - h5 for font size, right? RRRRIIIIIIGGGHHHHHTTTT?

File: data/vocabularies.js [hunk 5]
```diff
@@ -72,7 +63,6 @@ export const vocabularies = [
     translation: "Marvelous",
```
Comment (CultOfTheDeadCrow) [inline data/vocabularies.js:63]: What a splendid word!

**Review by CultOfTheDeadCrow — APPROVED** [review]
lgtm!

**Pull request merged**